### ISSUE TITLE
it can't generate events from this.$emit(CLICK_EVENT) 

### ISSUE THREAD

**Issue by cuijing1031** [vuese/vuese#11]
There is a coding rule in our team that event names are defined by const variables in upper case。The code is like:

```js
const EVENT_CLOSE = 'close'
// ...
this.$emit(EVENT_CLOSE, e)
```

But in this case,  the result of `bt.isStringLiteral(args[0])`  is false

```js
if (args.length && bt.isStringLiteral(args[0])) {
     result.name = args[0].value;
}
```
so the events block is empty in the markdown file.

**Comment by HcySunYang** [comment]
Um... I think about how to improve

**Comment by HcySunYang** [comment]
Which one has a better idea?

**Comment by roxburghm** [comment]
Run across this myself and it's fixed in my latest pull request #58 along with the addition of compute docs.

## Events


|Event Name|Description|Parameters|
|---|---|---|
|`PASS_CALLBACK`|an event that uses a constant for the callback|-|
|hiya|an event that uses a string literal for the callback|-|
